README.md

```diff
@@ -263,9 +253,9 @@ Great job! :sparkles:
 
 ### Stretch goals
 
-* Refactor the children of both class components into presentational functional components.
-* More CSS
-* Create a form input to dynamically create a Github user card from any username
+- Refactor the children of both class components into presentational functional components.
```

Comment:
No more classes so this should probably change!

Reply:
Ah good catch

Reply:
```suggestion
- Refactor the children of both stateful components into separate files and pass the state values down to them
```